pyvista/core/filters/poly_data.py

```diff
@@ -2054,7 +2054,9 @@ def geodesic(
         output["vtkOriginalPointIds"] = original_ids
 
         # Do not copy textures from input
-        output.clear_textures()
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
```

Comment:
I recommend only ignoring our own deprecation warning here. Better not have false negatives.